tests/unit/controllers/pull-requests-test.js

```diff
@@ -112,25 +112,14 @@ module('Unit | Controller | pull-requests', function(hooks) {
       },
     ]);
 
-    assert.deepEqual(
-      users,
-      [
-        {
-          handle: '@emberjs',
-          profileLink: 'https://github.com/emberjs'
-        },
+    const hasUsers = users.some( user => user['handle'] === '@zoey' ) 
+      && users.some( user => user['handle'] === '@tomster' ) 
+      && users.some( user => user['handle'] === '@emberjs' );
 
-        {
-          handle: '@tomster',
-          profileLink: 'https://github.com/tomster'
-        },
-
-        {
-          handle: '@zoey',
-          profileLink: 'https://github.com/zoey'
-        },
-      ],
-      'We can sort users by their handle.'
+    assert.strictEqual(
+      users.length === 3 && hasUsers === true,
```

Comment:
In general, I try to avoid writing a compound expression for an assertion. If an expression fails, it's hard to tell which subexpression failed.

I think the test itself (assert that random ordering worked) was difficult to do, so let's leave your assertion in for now. 👍